examples/plot_raster_stack_crop.py

```diff
@@ -0,0 +1,135 @@
+"""
+Stack and Crop Rasters with EarthPy
+====================================
+
+Learn how to stack and crop satellite imagery with GeoPandas objects in EarthPy
+"""
+
+###############################################################################
+# Stacking and Cropping Rasters using EarthPy
+# ---------------------------------------------
+#
+# .. note:: The examples below will show you how to use the ``es.stack()`` and ``es.crop_image()`` functions.
+
+
+###############################################################################
+# Stacking Multi Band Imagery
+# -----------------------------
+#
+# Let's explore a simple raster stack using EarthPy. To begin, import the needed packages
+# and create an array to be plotted. Below we plot the data as continuous with a colorbar
+# using the ``plot_bands()`` function.
+
+###############################################################################
+# Import Packages
+# ------------------------------
+#
+# To begin, import the needed packages. You will primarily be using the spatial module
+# of EarthPy, but other packages will be needed as well.
+
+import os
+from glob import glob
+import matplotlib.pyplot as plt
+import rasterio as rio
+from rasterio.plot import plotting_extent
+import earthpy as et
+import geopandas as gpd
+import earthpy.spatial as es
+import earthpy.plot as ep
+
+
+###############################################################################
+# Get Example Data Ready for Stack
+# ----------------------------------
+# Create a stack from all of the Landsat .tif files (one per band) with the
+# `es.stack()` function. Make sure you output a raster with the `out_path`
+# argument, as it will be needed for the crop.
+
+# Get data and set your home working directory
+
+data = et.data.get_data("cs-test-landsat")
+data2 = et.data.get_data("cold-springs-fire")
+os.chdir(os.path.join(et.io.HOME, "earth-analytics"))
+os.environ[
+    "PROJ_LIB"
+] = r"C:\Users\Nathan\Anaconda3\envs\earth-analytics-python\Library\share"
+
+# Opening up boundary layer to use for the cropping
+
+boundary = gpd.read_file(
+    "data/cold-springs-fire/vector_layers/fire-boundary-geomac/co_cold_springs_20160711_2200_dd83.shp"
+)
+
+# Stacking the landsat bands needed
+
+landsat_bands = "data/cs-test-landsat/LC08_L1TP_034032_20160621_20170221_01_T1_sr_band*[2-4]*.tif"
+stack_bands = glob(landsat_bands)
+stack_bands.sort()
+datafile = os.path.join("data", "outputs")
+if os.path.isdir(datafile) == False:
+    os.mkdir(datafile)
+# Creating an output path for the raster
+
+raster = os.path.join(datafile, "raster.tiff")
+arr, rast = es.stack(stack_bands, out_path=raster, nodata=-9999)
+
+# Changing the Coordinate Reference System of the boundary to match that of the raster.
+
+boundary = boundary.to_crs(rast["crs"])
```

Comment:
Given this is reprojected how about `crop_bound_crs`
i can't remember what CRS you projected to here so change the crs part above to the new projection so if it's WGS84 id use that instead of crs.